### PR TITLE
Add connect.keep.alive.interval.ms configuration option

### PR DESCRIPTION
**Pull request by vuckooo**
Related to issue: https://issues.jboss.org/browse/DBZ-670

### CODE REVIEW

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java [hunk 1]
```diff
@@ -709,6 +709,15 @@ public static EventProcessingFailureHandlingMode parse(String value) {
                                                 .withDefault(true)
                                                 .withValidation(Field::isBoolean);
 
+    public static final Field KEEP_ALIVE_INTERVAL_MS = Field.create("connect.keep.alive.interval.ms")
+                                                            .withDisplayName("Keep alive interval (ms)")
+                                                            .withType(Type.INT)
+                                                            .withWidth(Width.SHORT)
+                                                            .withImportance(Importance.LOW)
+                                                            .withDescription("Interval in milliseconds to wait for connection checking if keep alive thread is used.")
+                                                            .withDefault(60 * 1000)
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java]: Super-small nitpick, I'd suggest to specify the default as `Duration.ofMinutes(1).toMillis()` which I think makes this easier to read, esp. as it puts the unit in there. Leaving it to you whether you want to change it or not.

Comment (vuckooo) [reply debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java]: No problem, I saw the same syntax few lines above, but it's cleaner like you proposed, so I'll change it :)

Comment (vuckooo) [reply debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java]: I'll change the docs, did't find them at first

**Comment by gunnarmorling** [comment]
Rebased and applied. Thanks a lot, @vuckooo!

**Pull request closed without merging**